.github/renovate.json5

```diff
@@ -655,8 +655,16 @@
     },
     {
       "matchPackageNames": [
-        "github.com/aws/aws-sdk-go-v2/service/ec2",
+        "github.com/Azure/azure-sdk-for-go/sdk/azcore",
+        "github.com/Azure/azure-sdk-for-go/sdk/azidentity",
+        "github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/compute/armcompute/v6",
+        "github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/network/armnetwork/v6",
         "github.com/aliyun/alibaba-cloud-sdk-go",
+        "github.com/aws/aws-sdk-go-v2",
+        "github.com/aws/aws-sdk-go-v2/config",
+        "github.com/aws/aws-sdk-go-v2/feature/ec2/imds",
+        "github.com/aws/aws-sdk-go-v2/service/ec2",
+        "github.com/aws/smithy-go",
         "renovatebot/github-action"
       ],
```

Comment:
I would suggest to do something like this so that we don't have do add a new path every time it changes.
```
"matchDepPatterns": [
   "github.com/Azure/azure-sdk-for-go/*",
   "github.com/aws/aws-sdk-go-v2/*",
]

```

Reply:
updated

Reply:
@HadrienPatte notice that I mentioned `matchDepPatterns` and not `matchPackageNames`. We need to have both in this scetion.

Reply:
Ah sorry I missed that, looking a bit into it, it looks like there's also a `matchPackagePatterns` attribute used in a similar block [here](https://github.com/cilium/cilium/blob/d89d1414a87de6aaf107af2876f9c0107945aad0/.github/renovate.json5#L237). Do you recommend to use `matchPackagePatterns` or `matchDepPatterns`?

Reply:
We also use `matchDepPatterns` [here](https://github.com/cilium/cilium/blob/d89d1414a87de6aaf107af2876f9c0107945aad0/.github/renovate.json5#L580). I would go with this one.

Reply:
Thanks, pushed an update, let me know if that looks good ;)